test/unit/Reflection/ReflectionFunctionAbstractTest.php

```diff
@@ -349,6 +349,7 @@ public function returnTypeFunctionProvider()
             ['returnsString', 'string'],
             ['returnsNull', 'null'],
             ['returnsObject', '\stdClass'],
+            ['returnsVoid', 'void'],
```

Comment:
Oh I see where your confusion may have arisen; `returnsVoid` is the name of the function to be reflected in the test. The test is testing the `ReflectionFunctionAbstract#getReturnType()` method, and we expect, when cast to `string`, the value to be `void`.